### ISSUE TITLE
RadCalendar no longer provides eventData.color on inlineEventSelected trigger

### ISSUE THREAD

**Issue by Jabronious** [telerik/nativescript-ui-feedback#1051]
> Please take a minute to read our [NativeScript Code of Conduct](https://github.com/NativeScript/codeofconduct) before proceeding with posting issues or discussing. The purpose of this guide is to make communication and cooperation within our forums a pleasure for you and the other members.

### Did you verify this is a real problem by searching the [NativeScript Forum](https://discourse.nativescript.org/)?
Yes

### Tell us about the problem
When an inlineEventSelected event is triggered the args.eventData.color is not correct on android (it is set to #B233B5E5) and on iOS I receive null. On the triggered event I loop through all given events and log the colors of each so I know that they are being stored initially correct.

### Which platform(s) does your issue occur on?
Both

### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** not available?
- **CLI:** 5.2.2
- **Cross-platform modules:** 5.1.0
- **Runtime(s):** tns-android/tns-ios - 5.1.0

### Please tell us how to recreate the issue in as much detail as possible.
Populate a basic RadCalendar with at least one event and include inlineEventSelected trigger. In the function called from that trigger include a console log to display the event data color.

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
Basically, when you create a RadCalendar you would need to include the 
`(inlineEventSelected)="onEventSelected($event)"`
then:
calendar.component.ts
```
...
onEventSelected(args) {
    console.log(args.eventData.eventColor); // this will equal null on ios and #B233B5E5 on android
}
```
